File: Java_Programs-master/Factorial.java
public class Factorial
{
    public static void main(String [] args)
    {
        int number = 5;
        int fact = 1;
        int i;
        for(i = 1; i <= number; i++)
        {
            fact = fact * i;
        }
        System.out.println("\nFactorial of " + number + " is : " + fact);
    }
}

File: Algorithms-in-C/number_theory/factorial.c
//Factorial of given number using recursion

#include <stdio.h>
#include <conio.h>
 
int factorial(int);
 
int main()
{
    int num;
    int result;
 
    printf("Enter a number to find it's Factorial: ");
    scanf("%d", &num);
    if (num < 0)
    {
        printf("Factorial of negative number not possible\n");
    }
    else
    {
        result = factorial(num);
        printf("The Factorial of %d is %d.\n", num, result);
    }
    return 0;
}
int factorial(int num)
{
    if (num == 0 || num == 1)
    {
        return 1;
    }
    else
    {
        return(num * factorial(num - 1));
    }
}
